### PR TITLE
clang-format sources of centrality

### PR DESCRIPTION
**Pull request by angriman**
This is a continuation of #803

### CODE REVIEW

File: networkit/cpp/centrality/DynBetweenness.cpp [hunk 1]
```diff
@@ -215,59 +220,61 @@ void DynBetweenness::update(GraphEvent event) {
...
 
-        while (! stack.empty()) {
+        while (!stack.empty()) {
             node y = stack.top();
             if (!visited[y]) {
-                std::priority_queue<std::pair<double, node>, std::vector<std::pair<double,node>>, CompareDist> Qnew;
-                std::priority_queue<std::pair<double, node>, std::vector<std::pair<double,node>>, CompareDist> Qold;
+                std::priority_queue<std::pair<double, node>, std::vector<std::pair<double, node>>,
+                                    CompareDist>
+                    Qnew;
```
Comment (fabratu) [inline networkit/cpp/centrality/DynBetweenness.cpp:228]: That looks a bit strange. Is it possible to change this edit?

File: networkit/cpp/centrality/DynBetweenness.cpp [hunk 2]
```diff
@@ -215,59 +220,61 @@ void DynBetweenness::update(GraphEvent event) {
...
+                std::priority_queue<std::pair<double, node>, std::vector<std::pair<double, node>>,
+                                    CompareDist>
+                    Qold;
```
Comment (fabratu) [inline networkit/cpp/centrality/DynBetweenness.cpp:231]: As above.

File: networkit/cpp/centrality/DynBetweenness.cpp (hunk 1 above)
Comment (angriman) [reply networkit/cpp/centrality/DynBetweenness.cpp:228]: One possibility is to have a shorter type name for the priority queue, e.g., `using PrioQ = std::priority_queue<...>;`.

**Review by fabratu — APPROVED** [review]
LGTM now.

**Review by fabratu — APPROVED** [review]

**Pull request merged**